context/app/static/js/components/DetailProtocols.jsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import styled from 'styled-components';
+import Typography from '@material-ui/core/Typography';
+import Link from '@material-ui/core/Link';
+import Divider from '@material-ui/core/Divider';
+import Paper from '@material-ui/core/Paper';
+import SectionHeader from './SectionHeader';
+import SectionItem from './SectionItem';
+import SectionContainer from './SectionContainer';
+
+const StyledTypography = styled(Typography)`
+  margin: 2px 0px 2px 0px;
+`;
+
+const StyledLink = styled(Link)`
+  color: #3781D1;
+`;
+
+const StyledPaper = styled(Paper)`
+  padding: 20px 40px 20px 40px;
+`;
+
+function ProtocolLink(props) {
+  const { protocolUrl: url } = props;
```

Comment:
The "protocolUrl" we get from the API just starts with the hostname? ARGH. Please file an issue with search-api, and let's have a longer variable name (`urlMinusProtocol`?) so there's no confusion about what it contains.

Reply:
A bit confusing given we're linking to 'protocols' but sure, I'll make the change.